README.md

```diff
@@ -69,6 +69,9 @@ Learn more on parametrization in the [**documentation**](https://facebookresearc
 
 Check out our [**documentation**](https://facebookresearch.github.io/nevergrad/)! It's still a work in progress, don't hesitate to submit issues and/or PR to update it and make it clearer!
 
+## Windows installation
+
+For windows installation, please refer to [the windows documention](docs/windows.md)
```

Comment:
Most of the install intructions are in getting started, you should at least add a link there as well.
I would tend to remove the lines here and update line 14 to state that Windows install can be found there

Reply:
I have reduced all images, it was quite ugly (not sure this is beautiful now, I think I am not good at this :) ).
I have corrected the links, as you suggested. I am not familiar to rst, so I just add a 

```
Installing on Windows
----------

For windows installation, please refer to `the windows documention <https://github.com/facebookresearch/nevergrad/blob/master/docs/windows.md>`
```
in the getting_started.rst file. Not sure if this is a good way to do it :)
(and I can't check the link)